transcom/mymove
# INT-B-19148

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A922834)

## Summary

BLUEBARK Moves should receive NO emails at all. 

### How to test

1. Run the backend with `EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run`
2. Create a new customer and create a BLUEBARK move
3. Approve the move as a SC
4. Approve items as TOO
5. Make changes as Prime
6. Approve/reject as TIO

Make sure no emails were sent
